ci/container_index/lib/checks/value_validation.py

```diff
@@ -0,0 +1,33 @@
+from os import path
+
+from ci.container_index.lib.checks.basevalidation import \
+    OptionalClonedValidator
+
+
+class GitCloneValidator(OptionalClonedValidator):
```

Comment:
I see the class names are making sense, it would also make more sense if we add few statements as docstrings.

Reply:
Added